vault/resource_pki_secret_backend_role_test.go

```diff
@@ -67,7 +76,36 @@ func TestPkiSecretBackendRole_basic(t *testing.T) {
 		CheckDestroy: testPkiSecretBackendRoleCheckDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testPkiSecretBackendRoleConfig_basic(name, backend, 3600, 7200),
+				Config: testPkiSecretBackendRoleConfig_basic(name, backend, 3600, 7200, legacyPolicyIdentifiers),
```

Comment:
Since we currently don't have access to the latest Vault builds within CI, the acceptance tests for 1.11 fields will unfortunately fail within GitHub Actions. At present, we're skipping 1.11 field tests until we can get the latest version of Vault running in CI :) 

We use `testutil.SkipTestEnvSet(t, testutil.EnvVarSkipVaultNext)` to do so. (see example [here](https://github.com/hashicorp/terraform-provider-vault/blob/release/vault-next/vault/resource_gcp_auth_backend_test.go#L30))

Reply:
:+1: